p/scripts/user.query.js

```diff
@@ -11,27 +11,63 @@ function init_draggable() {
 		return;
 	}
 	
-	let source;
+	var source;
 	const configureQueries = document.querySelector('#configureQueries');
+	const addMarker = function(position, element) {
+		const hr = configureQueries.querySelector('hr.drag-drop-marker');
+		if (null === hr) {
+			element.insertAdjacentHTML(position, '<hr class="drag-drop-marker"/>');
+		}
+	}
+	const removeMarker = function() {
+		const hr = configureQueries.querySelector('hr.drag-drop-marker');
+		if (null !== hr) {
+			hr.remove();
+		}
+	}
```

Comment:
```suggestion
	};
```

Reply:
A semicolon is also missing line 77 from previous PR

Reply:
PS That's why I set up jshint after all. ^_^

Reply:
Indeed; and I probably have merged too fast...